src/compiler/class/ClassDeclaration.ts

```diff
@@ -906,6 +907,23 @@ export class ClassDeclaration extends ClassDeclarationBase<ts.ClassDeclaration>
         return classes;
     }
 
+    /**
+     * Gets the structure equivalent to this node.
+     */
+    getStructure(): ClassDeclarationStructure {
+        const getExtends = this.getExtends();
+        return callBaseGetStructure<ClassDeclarationSpecificStructure>(ClassDeclarationBase.prototype, this, {
+            ctors: this.getConstructors().filter(ctor => !ctor.isOverload())
+                .map(ctor => ctor.getStructure()),
+            methods: this.getMethods().filter(method => !method.isOverload() || method.isAbstract())
```

Comment:
Heh, this is a bug in the library. `isOverload()` should return false for abstract members.